suite-native/module-connect-popup/src/components/ButtonRequestsOverlay.tsx

```diff
@@ -0,0 +1,45 @@
+import { useSelector } from 'react-redux';
+
+import { selectDevice } from '@suite-common/wallet-core';
+import { Box, Text, VStack } from '@suite-native/atoms';
+import { ConfirmOnTrezorImage } from '@suite-native/device';
+import { ButtonRequest } from '@suite-common/suite-types';
+import { Translation } from '@suite-native/intl';
+
+export const ButtonRequestsOverlay = () => {
+    const selectedDevice = useSelector(selectDevice);
+
+    if (!selectedDevice?.buttonRequests || selectedDevice.buttonRequests.length === 0) {
+        return null;
+    }
+
+    const getButtonRequestComponent = (request: ButtonRequest) => {
+        if (request.code === 'ButtonRequest_Address') {
+            return (
+                <Box paddingBottom="extraLarge" key={request.code}>
```

Comment:
nit: `request.code` may not be unique, there can be multiple button request with the same code on the device...